includes/definitions/discovery/poweralert.yaml

```diff
@@ -4,9 +4,41 @@ modules:
         features: .1.3.6.1.4.1.850.10.2.2.1.9.2
         hardware: .1.3.6.1.2.1.33.1.1.2.0
         serial: .1.3.6.1.4.1.850.100.1.1.2.0
+        version: .1.3.6.1.2.1.33.1.1.4.0
```

Comment:
Think you need an array when you hade multiple. Look at other yamls for examples